packages/repository-tests/src/crud/relations/acceptance/has-many-inclusion-resolver.relation.acceptance.ts

```diff
@@ -0,0 +1,273 @@
+          });
+          const result = await customerRepo.find({
+            include: [{relation: 'orders'}],
+          });
+
+          const expected = {
+            id: c1.id,
+            name: 'c1',
+            orders: [
+              {
+                id: o1.id,
+                description: 'order from c1',
+                customerId: c1.id,
+                isShipped: features.emptyValue,
+                // eslint-disable-next-line @typescript-eslint/camelcase
+                shipment_id: features.emptyValue,
+              },
+            ],
+            parentId: features.emptyValue,
+          };
+          expect(toJSON(result)).to.deepEqual([toJSON(expected)]);
+        });
+
+        it('returns related instances to target models via find() method', async () => {
+          const c1 = await customerRepo.create({name: 'Thor'});
+          const c2 = await customerRepo.create({name: 'Hella'});
+          const o1 = await orderRepo.create({
+            customerId: c1.id,
+            description: 'Mjolnir',
+          });
+          const o2 = await orderRepo.create({
+            customerId: c1.id,
+            description: 'Pizza',
+          });
+          const o3 = await orderRepo.create({
+            customerId: c2.id,
+            description: 'Blade',
+          });
+
+          const result = await customerRepo.find({
+            include: [{relation: 'orders'}],
+          });
+
+          const expected = [
+            {
+              id: c1.id,
+              name: 'Thor',
+              orders: [
+                {
+                  id: o1.id,
+                  description: 'Mjolnir',
+                  customerId: c1.id,
+                  isShipped: features.emptyValue,
+                  // eslint-disable-next-line @typescript-eslint/camelcase
+                  shipment_id: features.emptyValue,
+                },
+                {
+                  id: o2.id,
+                  description: 'Pizza',
+                  customerId: c1.id,
+                  isShipped: features.emptyValue,
+                  // eslint-disable-next-line @typescript-eslint/camelcase
+                  shipment_id: features.emptyValue,
+                },
+              ],
+              parentId: features.emptyValue,
+            },
+            {
+              id: c2.id,
+              name: 'Hella',
+              orders: [
+                {
+                  id: o3.id,
+                  description: 'Blade',
+                  customerId: c2.id,
+                  isShipped: features.emptyValue,
+                  // eslint-disable-next-line @typescript-eslint/camelcase
+                  shipment_id: features.emptyValue,
+                },
+              ],
+              parentId: features.emptyValue,
+            },
+          ];
+          expect(toJSON(result)).to.deepEqual(toJSON(expected));
+        });
+
+        it('returns related instances to target models via findById() method', async () => {
```

Comment:
Please introduce a new `describe` block to group `findById` tests and apply the comments I posted for `find` tests.

Reply:
I removed `find*` from all titles as my intention was to show different use cases instead of testing `find*` here.